types/stripe-node/index.d.ts

```diff
@@ -4238,14 +4241,29 @@ declare namespace StripeNode {
 
         interface ISubscriptionListOptions extends IListOptionsCreated {
             /**
+             * The billing mode of the subscriptions to retrieve.
+             */
+            billing?: "charge_automatically" | "send_invoice";
```

Comment:
I don't see this property in the documentation. Am I missing something?

Reply:
Which documentation are you reading? It seems to be available on their website, here: https://stripe.com/docs/api#list_subscriptions-billing

Reply:
Ah, it's a preview feature. Must have looking at docs without preview features being displayed. All good then 👍